### DISCUSSION TITLE
Style for rating in content

### DISCUSSION THREAD

**Discussion by Bepo2902** [e107inc/e107#5046] [Q&A Themes]
I have already posted this in Gitter, but maybe here would be the better place.
I am using the build in Bootstrap 5 theme with the Slate skin, if this is any important.
I have created some article in the content function and offered the user to rate my articles, but the the rating option is stuck to the right like in the screenshot below.

![image](https://github.com/e107inc/e107/assets/46347624/e86ee459-c4ce-4604-a719-0c7d3e36a924)

I would like to have it like in the screenshot below.

![image](https://github.com/e107inc/e107/assets/46347624/2ffb47ec-8201-4d0d-903e-9c6ef3b488e3)

I found the bit in the developertools of the browser, but there I can only change temporarily. See screenshot below. 

![image](https://github.com/e107inc/e107/assets/46347624/ab9e1cc4-8b00-4dd3-aa64-55dc46c88f1d)

I need to find either the actual file because I think the style I need to change is inline, or I need the correct CSS file.
Can someone help, please?
Thanks in advance.

**Comment by Bepo2902** [comment]
Jimmi08 answered this question in Gitter.
I had to add the following to the style.css

div.e-rate {
width: 300px!important;
}

It worked fine, but I would also like to make some more space between "Rate this page" and the stars, as seen in screenshot below.

![image](https://github.com/e107inc/e107/assets/46347624/48558da9-2f47-4060-b93d-6c388b8e623f)
![image](https://github.com/e107inc/e107/assets/46347624/ee4c6fe4-5aaf-4d48-afe0-cc03f9b7e850)

Thank you

**Reply by tgtje** [reply]
Dirty fix @Bepo2902 ; **_you will loose_** the alterations when an update happens.
Add a blank space where needed..
e107_web/js/core/front.jquery.js (438)
‎e107_languages/English/lan_rate.php (15)
e107_languages/English/lan_page.php (14)
